lib/drivers.js

```diff
@@ -0,0 +1,157 @@
+import path from 'path';
+import _ from 'lodash';
+import log from './logger';
+
+// TODO the location of the driver data file should be configurable
+const DRIVERS_JSON = path.resolve(__dirname, '..', '..', '.drivers.json');
```

Comment:
I was thinking about this. If someone reinstalls appium, then all the drivers need to also be reinstalled. But if .drivers.json lives in the home folder, it will be out of sync. Thoughts on this design?

Reply:
When the Appium server starts, should it verify that what's in .drivers.json matches what's installed in node_modules? We could then have two new driver commands which are mentioned in an error message:

```
appium driver verify  # prints out errors if anything in .drivers.json is not actually installed
appium driver reinstall  # reinstalls any that don't exist
```

On this kind of model it would make sense to have .drivers.json in $HOME, maybe should be called `.appium-drivers.json`, or we have a dir `.appium/drivers.json`, in case we need more stuff there in the future.

Reply:
in case with Appium Desktop we usually cannot write into its installation folder

Reply:
Yeah, good point. We should have a `.appium` dir which can host `.appium/drivers.json` and `.appium/plugins.json` and maybe the driver npm modules themselves. (see forthcoming comment)

Reply:
@mykola-mokhnach I had the thought that we could just keep all the installed modules in a location separate from appium, and then simply `require` that full path rather than the bare module name. This is also nice because when people reinstall Appium, their drivers don't get blown away.

Reply:
This seems to work well in practice!